# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../python/libdir-from-configure.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


Fix pythons' Makefile template to take the bin and libdir from configure -
fixes to use lib64 for architectures required.

  - Rene Rebe <rene@exactcode.de>

--- Python-2.4.2/Makefile.pre.in	2005-03-29 01:23:01.000000000 +0200
+++ Python-2.4.2-fixed/Makefile.pre.in	2006-02-05 11:51:57.403019250 +0100
@@ -78,8 +78,8 @@
 exec_prefix=	@exec_prefix@
 
 # Expanded directories
-BINDIR=		$(exec_prefix)/bin
-LIBDIR=		$(exec_prefix)/lib
+BINDIR=		@bindir@
+LIBDIR=		@libdir@
 MANDIR=		@mandir@
 INCLUDEDIR=	@includedir@
 CONFINCLUDEDIR=	$(exec_prefix)/include
